src/map/trade_container.h

```diff
@@ -36,8 +36,9 @@
 
 enum CRAFT_TYPE
 {
-    CRAFT_SYNTHESIS   = 0,
-    CRAFT_DESYNTHESIS = 1,
+    CRAFT_SYNTHESIS        = 0,
+    CRAFT_DESYNTHESIS      = 1,
+    CRAFT_NO_MATERIAL_LOSS = 2,
```

Comment:
I wouldn't class this as an entirely new type of craft, so I'd probably call this `CRAFT_SYNTHESIS_NO_MATERIAL_LOSS` - even if it is super long

Reply:
How about `CRAFT_SYNTHESIS_NO_LOSS` ?